build/Build.Settings

```diff
@@ -20,6 +20,11 @@
    <PropertyGroup Condition="'$(Configuration)' == 'Release'">
      <UseDebugLibraries>false</UseDebugLibraries>
      <WholeProgramOptimization>true</WholeProgramOptimization>
+     <CodeAnalysisRuleSet>..\DefaultRules.ruleset</CodeAnalysisRuleSet>
+     <RunCodeAnalysis>false</RunCodeAnalysis>
+     <!-- Do not enable code analysis on old compiler versions -->
+     <RunCodeAnalysis Condition="'$(VCToolsRedistVersion)' != '' AND '$(VCToolsRedistVersion)' >= 14.15.26706">true</RunCodeAnalysis>
```

Comment:
The only build this failed on before was the release build: https://ci3.dot.net/job/aspnet_IISIntegration/job/release_2.2/job/windows-Configuration_Release_prtest/669/console. Maybe you missed a flag for release?

Reply:
Which flag?

Reply:
Nvrm, didn't realize the build isn't run for debug.